azkaban_cli/azkaban_cli.py

```diff
@@ -445,7 +445,7 @@ def __resume_flow_execution(ctx, execution_id):
 
     try:
         json = azkaban.resume_flow_execution(execution_id)
-        __log_flow_execution_updates(json)
+        __log_flow_execution_updates('Flow succesfully resumed')
```

Comment:
This won't work though, since the function expects a json-like (dictionary) object. I would just put a `logging.info('Flow successfully resumed')` for example

Reply:
done